node/src/components/block_proposer/tests.rs

```diff
@@ -585,6 +567,8 @@ fn test_proposer_with(
     let block = proposer.propose_proto_block(config, test_time, past_deploys, true);
     let all_deploys = BlockLike::deploys(&block);
     proposer.finalized_deploys(all_deploys.iter().map(|hash| **hash));
+    println!("proposed deploys {}", block.wasm_deploys().len());
+    println!("proposed transfers {}", block.transfers().len());
```

Comment:
Leftover?

Reply:
I'd like to keep these, as they are nice to have as output in the tests if they fail.

Reply:
Then change the logic to only print them when it fails, please.